### PR TITLE
Add Pester test for Dotnet TLS

### PR DESCRIPTION
**Pull request by vsafonkin**
# Description
Added Pester test

#### Related issue:
[#805](https://github.com/actions/virtual-environments-internal/issues/805)

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Tests/Tools.Tests.ps1 [hunk 1]
```diff
@@ -14,4 +14,12 @@ Describe "Kind" {
     It "Kind" {
         "kind version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "DotnetTLS" {
+    It "Tls 1.2 is enabled" {
+        $protocols = [Net.ServicePointManager]::SecurityProtocol
+        $protocolArr = $protocols -split ', '
+        $protocolArr.Contains('Tls12') | Should -BeTrue
```
Comment (maxim-lobanov) [inline images/win/scripts/Tests/Tools.Tests.ps1:23]: May be we could simplify this check?
```
[Net.ServicePointManager]::SecurityProtocol | Should -BeLike "Tls12"
```

Comment (al-cheb) [inline images/win/scripts/Tests/Tools.Tests.ps1:23]: [Net.ServicePointManager]::SecurityProtocol -band "Tls12" | Should -Be Tls12

Comment (maxim-lobanov) [reply images/win/scripts/Tests/Tools.Tests.ps1:23]: Please ignore mine message, @al-cheb's proposal is better

Comment (vsafonkin) [reply images/win/scripts/Tests/Tools.Tests.ps1]: Done, thanks!

**Pull request merged**